napalm/nxos_ssh/nxos_ssh.py

```diff
@@ -453,21 +453,15 @@ def _send_command(self, command, raw_text=False, cmd_verify=True):
         """
         return self.device.send_command(command, cmd_verify=cmd_verify)
 
-    def _send_command_list(self, commands, expect_string=None):
-        """Wrapper for Netmiko's send_command method (for list of commands."""
-        output = ""
-        for command in commands:
-            output += self.device.send_command(
-                command,
-                strip_prompt=False,
-                strip_command=False,
-                expect_string=expect_string,
-            )
-        return output
+    def _send_command_list(self, commands, expect_string=None, **kwargs):
+        """Send a list of commands using Netmiko"""
+        return self.device.send_multiline(
+            commands, expect_string=expect_string, **kwargs
+        )
 
     def _send_config(self, commands):
         if isinstance(commands, str):
-            commands = (command for command in commands.splitlines() if command)
+            commands = [command for command in commands.splitlines() if command]
```

Comment:
I converted this from a generator expression to a list comprehension as the generator expression doesn't work if you try to iterate over it a second time (which can cause problems and make debugging confusing).